app/controllers/chat/channel.js

```diff
@@ -1,6 +1,10 @@
 import Ember from 'ember';
 
 export default Ember.Controller.extend({
+  canParticipate: Ember.computed.readOnly('session.isAuthenticated'),
+
+  cannotParticipate: Ember.computed.not('canParticipate'),
```

Comment:
what do you think about this being a computed property on the session itself? maybe like a <code>hasWriteAccess</code>?